docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -206,24 +204,14 @@ Preparing For a Release
       git commit -am "Updated Vagrantfile"
       git push --set-upstream origin release/flocker-${VERSION}
 
-#. Set up Google Cloud Storage and Amazon S3 credentials:
-
-   Creating the Vagrant machine attempts to copy the ``~/.boto`` configuration file from the host machine.
+#. Set up ``AWS Access Key ID`` and ``AWS Secret Access Key`` Amazon S3 credentials:
 
-   Run:
+   Creating the Vagrant machine attempts to copy the ``~/.aws`` configuration directory from the host machine.
+   This means that ``awsutil`` may have correct defaults.
```

Comment:
Where does the name `awsutil` come from?  Does this refer to the `aws` command line tool being introduced here?  As far as I can tell that's provided by a package called `awscli`.